src/main/java/com/google/cloud/resourcemanager/ResourceManager.java

```diff
@@ -337,4 +343,76 @@ public static ProjectListOption fields(ProjectField... fields) {
    *     Platform Services</a>
    */
   List<Boolean> testPermissions(String projectId, List<String> permissions);
+
+  /**
+   * Clears a Policy from a resource.
+   *
+   * @throws ResourceManagerException upon failure
+   * @see <a href=
+   *     "https://cloud.google.com/resource-manager/reference/rest/v1/folders/clearOrgPolicy">Resource
+   *     Manager clearOrgPolicy</a>
+   */
+  void clearOrgPolicy(String resource, ClearOrgPolicyRequest request);
+
+  /**
+   * Gets the effective Policy on a resource
+   *
+   * <p>This is the result of merging Policies in the resource hierarchy. The returned Policy will
+   * not have an etag set because it is a computed Policy across multiple resources. Subtrees of
+   * Resource Manager resource hierarchy with 'under:' prefix will not be expanded.
```

Comment:
will not be --> are not
per Google developer writing guidelines that recommend present tense

Reply:
done